integration_tests/service.go

```diff
@@ -5,7 +5,9 @@ import (
 	"time"
 )
 
-type RealCounterService struct{}
+type RealCounterService struct {
+	UnimplementedCounterServiceServer
```

Comment:
not sure why do you need this here

Reply:
golangci-lint was giving a warning, and hence I added it there, will remove it 🙇🏼